_config.yml

```diff
@@ -644,97 +644,10 @@ navigation:
         position: 3
     "*radspreadprocessing":
         title: RadSpreadProcessing
-    controls/radspreadprocessing/working-with-worksheets:
```

Comment:
Many removed titles? Is this change correct?

Reply:
Yes